tokens/sft.svg

```diff
@@ -0,0 +1 @@
81.695-.881-.215 0-.504.182-.642.406m-117.757 4.17c.528.432 1.133.787 1.344.789.412.003-.283-.53-1.537-1.177-.51-.264-.446-.134.193.388m2.689 1.379c.529.303 1.134.55 1.345.55.211 0-.048-.247-.576-.55-.529-.302-1.134-.549-1.345-.549-.211 0 .048.247.576.549m95.87 4.241c-.961.463-.993.516-.193.32 1.089-.266 2.313-.89 1.73-.881-.212.003-.903.255-1.537.561m-82.613.769c.634.29 1.498.524 1.921.519.423-.005-.01-.254-.961-.553-2.133-.672-2.476-.66-.96.034m79.731.178c-.792.253-.825.313-.192.34.422.018 1.027-.135 1.344-.34.674-.435.206-.435-1.152 0m-73.391 1.519c.317.215.908.393 1.313.396.983.008.676-.212-.737-.528-.842-.188-.997-.153-.576.132m68.203.018c-.484.208-.555.33-.192.33.317 0 .922-.149 1.345-.33.485-.208.556-.33.192-.33-.317 0-.922.148-1.345.33m-59.366 1.902c.96.326 3.345.53 3.042.26-.088-.078-1.024-.242-2.081-.364-1.296-.15-1.608-.116-.961.104m50.529.019c-1.036.264-1.055.294-.192.301.528.003 1.393-.132 1.921-.301 1.291-.414-.104-.414-1.729 0m-46.607.826c.274.173 1.448.304 2.611.291 2.553-.03 2.154-.233-.882-.448-1.242-.089-2.006-.019-1.729.157m41.996-.057-1.921.263 1.729.055c.951.029 1.988-.114 2.305-.318.317-.205.404-.348.192-.318l-2.305.318m-36.279.527c.088.078 1.024.242 2.081.364 1.296.15 1.608.116.961-.104-.96-.326-3.345-.53-3.042-.26m29.939.241c-1.021.215-1.113.29-.384.312.528.015 1.652-.115 2.497-.29 1.281-.265 1.345-.317.384-.311-.634.003-1.758.133-2.497.289m-19.105.667c1.01.08 2.567.079 3.458-.002.892-.081.066-.146-1.836-.144-1.902.001-2.632.067-1.622.146m8.826-.003c.793.083 2.09.083 2.882 0 .793-.083.144-.151-1.441-.151s-2.233.068-1.441.151" fill="#737dac"/><path d="M200.096 80.585c.687.086 1.811.086 2.498 0 .686-.085.125-.155-1.249-.155s-1.936.07-1.249.155m-79.708 67.926c.002 1.268.074 1.74.159 1.05.086-.691.084-1.728-.003-2.306-.088-.577-.158-.012-.156 1.256m103.789 27.858c0 4.332.056 6.105.125 3.938.068-2.166.068-5.71 0-7.877-.069-2.166-.125-.393-.125 3.939m-45.029 16.907c0 .739.079 1.042.175.672.097-.37.097-.975 0-1.345-.096-.37-.175-.067-.175.673m45.029 7.685c0 4.332.056 6.104.125 3.938.068-2.166.068-5.711 0-7.877-.069-2.166-.125-.394-.125 3.939m54.132 19.212c0 1.162.072 1.638.16 1.057a8.497 8.497 0 0 0 0-2.114c-.088-.581-.16-.105-.16 1.057m-.345 21.518c0 3.275.059 4.615.131 2.978a82.143 82.143 0 0 0 0-5.956c-.072-1.638-.131-.298-.131 2.978m-91.696 14.873c.478.092 1.169.088 1.537-.008.367-.096-.024-.171-.869-.167-.846.004-1.146.083-.668.175m-3.079 4.61a5.45 5.45 0 0 0 1.729 0c.476-.091.087-.166-.864-.166s-1.34.075-.865.166m26.326.005c.584.088 1.449.086 1.922-.006.472-.091-.006-.163-1.062-.16-1.057.003-1.444.077-.86.166m-17.68 1.161c1.215.076 3.203.076 4.419 0 1.215-.077.221-.139-2.21-.139-2.43 0-3.424.062-2.209.139m-6.349 42.644c.682.085 1.893.087 2.69.003.797-.083.239-.152-1.24-.154-1.48-.002-2.132.066-1.45.151m20.765.001c.691.086 1.728.084 2.305-.003.578-.088.013-.158-1.255-.156-1.268.002-1.741.074-1.05.159m-9.224.378c.478.092 1.169.088 1.537-.008.367-.096-.024-.171-.869-.167-.846.004-1.146.083-.668.175" fill="#8a93bc"/></g></svg>
```

Comment:
The icon is too big. Please make sure it's 5-7 kB max.

Reply:
Alright

I'll do that.

Reply:
I've made a new commit to the pull request.
Kindly check it out.